### PR TITLE
fix: revert accidental change to output format

### PR DESCRIPTION
**Pull request by jackgerrits**

### CODE REVIEW

File: vowpalwabbit/action_score.cc [hunk 1]
```diff
@@ -43,10 +43,10 @@ std::string to_string(const ACTION_SCORE::action_scores& action_scores_or_probs,
   std::string delim;
   for (const auto& item : action_scores_or_probs)
   {
-    ss << delim << fmt::format("{},{}", item.action, VW::fmt_float(item.score, decimal_precision));
+    ss << delim << fmt::format("{}:{}", item.action, VW::fmt_float(item.score, decimal_precision));
```
Comment (lalo) [inline vowpalwabbit/action_score.cc:46]: if all tests pass then we have missed a test

**Review by lalo — APPROVED** [review]

File: vowpalwabbit/action_score.cc (hunk 1 above)
Comment (jackgerrits) [reply vowpalwabbit/action_score.cc:46]: I agree, I don't think tests will pass though

Comment (jackgerrits) [reply vowpalwabbit/action_score.cc:46]: Actually, yep I think we're missing some tests here

**Review by ataymano — APPROVED** [review]

File: vowpalwabbit/action_score.cc (hunk 1 above)
Comment (jackgerrits) [reply vowpalwabbit/action_score.cc:46]: So it is covered by tests, but it seems like the harness is failing to see the diff here

**Pull request merged**